README.md

```diff
@@ -297,6 +297,21 @@ $ docker run -v /path/to/cache/dir:/data \
 	--htpasswd_file /etc/bazel-remote/htpasswd --max_size=5
 ```
 
+You can also enforce authentication with client certificates by passing in a `tls_ca_file`:
```

Comment:
Should we say something like "Instead of using a .htpasswd file, you can use client certificates..." ? I guess it doesn't make much sense to use both (even if it's possible).